packages/hash/frontend/src/blocks/page/PageSectionContainer.tsx

```diff
@@ -10,9 +11,9 @@ export const getPageSectionContainerStyles = (pageComments: PageThread[]) => {
   const commentsContainerWidth = pageComments?.length ? COMMENTS_WIDTH : 0;
 
   const paddingLeft = `max(calc((100% - ${
-    PAGE_CONTENT_WIDTH + commentsContainerWidth + PAGE_MIN_PADDING
+    PAGE_CONTENT_WIDTH + commentsContainerWidth
```

Comment:
@luisbettencourt this was adding additional padding to the left, was this intentional?

before
<img width="1248" alt="Screenshot 2022-11-04 at 09 20 18" src="https://user-images.githubusercontent.com/37743469/199939007-cb1ec24c-ce54-4836-a963-36adca15ba0b.png">


after
<img width="1251" alt="Screenshot 2022-11-04 at 09 20 02" src="https://user-images.githubusercontent.com/37743469/199938948-5dbfdc05-b52e-4c0e-9909-b8f50d83a05e.png">

Reply:
Nice spot @CiaranMn !

It wasn't intentional, it was only supposed to be applied when there are comments in the page.
I've made a suggestion here: https://github.com/hashintel/hash/pull/1340#discussion_r1013944672

Reply:
Understood, and thank you for the suggestion, applied!